e2e/run-suite

```diff
@@ -28,5 +28,5 @@ fi
 cd packages/grafana-e2e
 
 yarn $CMD --env BASE_URL=$URL,SLOWMO=$SLOWMO \
-  --config defaultCommandTimeout=30000,integrationFolder=../../e2e/$SUITE/specs,screenshotsFolder=../../e2e/$SUITE/screenshots,videosFolder=../../e2e/$SUITE/videos,fileServerFolder=./cypress,viewportWidth=1920,viewportHeight=1080,trashAssetsBeforeRuns=false \
+  --config defaultCommandTimeout=30000,integrationFolder=../../e2e/$SUITE/specs,screenshotsFolder=../../e2e/$SUITE/screenshots,videosFolder=../../e2e/$SUITE/videos,fileServerFolder=./cypress,viewportWidth=1920,viewportHeight=1080,trashAssetsBeforeRuns=false,videoUploadOnPasses=false \
```

Comment:
My thinking is that `videoUploadOnPasses=false` might speed up passing e2e tests a bit, but might just be me overthinking it 🤔

Reply:
Having a look at the docs: https://docs.cypress.io/guides/references/configuration#Videos and they mention
```
This only applies when recording your runs to the Dashboard. 
```
I'm not too familiar with the Cypress dashboard, but it sounds like this won't impact our build or local runs? Not sure though 🤔

Reply:
@vickyyyyyyy bad docs, it seems like it has effects on the time see https://github.com/grafana/grafana/pull/39573#issuecomment-925737787